django/contrib/admin/widgets.py

```diff
@@ -194,7 +195,22 @@ def label_for_value(self, value):
         key = self.rel.get_related_field().name
         try:
             obj = self.rel.model._default_manager.using(self.db).get(**{key: value})
-            return '&nbsp;<strong>%s</strong>' % escape(Truncator(obj).words(14, truncate='...'))
+            text = escape(Truncator(obj).words(14, truncate='...'))
```

Comment:
I'm not sure why the `ValueError` is being caught below. Removing it doesn't make any tests fails, so I'm not sure which line is meant to raise it.